rust/capture/src/config.rs

```diff
@@ -54,6 +54,10 @@ pub struct KafkaConfig {
     pub kafka_topic: String,
     #[envconfig(default = "events_plugin_ingestion_historical")]
     pub kafka_historical_topic: String,
+    #[envconfig(default = "events_plugin_ingestion")]
+    pub kafka_exceptions_topic: String,
+    #[envconfig(default = "events_plugin_ingestion")]
+    pub kafka_heatmaps_topic: String,
```

Comment:
btw I was thinking of on the plugin-server side to start with $$client_ingestion_warning as https://github.com/PostHog/posthog/blob/1bff1d16884846d9554f26056064c1ea2b567f34/plugin-server/src/worker/ingestion/event-pipeline/runner.ts#L208 is the easiest thing to pull into it's own pipeline and our caring degree about it is also rather low if we drop events